internal/cluster/cluster.go

```diff
@@ -431,32 +432,26 @@ func (m *Manager) tryAssignMachineCidrNonDHCPMode(cluster *common.Cluster) error
 	).Error
 }
 
-func (m *Manager) autoAssignMachineNetworkCidrs() error {
-	var clusters []*common.Cluster
+func (m *Manager) autoAssignMachineNetworkCidr(c *common.Cluster) error {
+	if !funk.ContainsString([]string{models.ClusterStatusPendingForInput, models.ClusterStatusInsufficient}, swag.StringValue(c.Status)) {
```

Comment:
i think we should have function that verifies if cluster is in one of the statuses ?

Reply:
Which one?

Reply:
i meant maybe worth having function as :
ifInStatus(cluster, listOfStatuses) ?

Reply:
And what's the difference between the function you mentioned and the function that is used now besides the name?  The fact that cluster is provided as parameter and not status ?

Reply:
not a big change agree, i think we have so many usages of this one so will be nice to have just function for it but agree it doesn't really help